cloudevent-broker/main.tf

```diff
@@ -13,3 +13,9 @@ resource "google_pubsub_topic" "this" {
   // TODO: Tune this and/or make it configurable?
   message_retention_duration = "600s"
 }
+
+module "topic-dashboard" {
+  source       = "../dashboard/topic"
+  project_id   = var.project_id
+  topic_prefix = var.name
+}
```

Comment:
How is this different from `dashboard.tf`? 🤔

Reply:
Oh it's not! This was an artifact of refactoring. Removed.